README.md

```diff
@@ -64,20 +68,84 @@ jobs:
           appstore_token: ${{ secrets.APPSTORE_TOKEN }}
           download_url: ${{ steps.attach_to_release.outputs.browser_download_url }}
           app_private_key: ${{ secrets.APP_PRIVATE_KEY }}
-          nightly: ${{ github.event.release.prerelease }}
+          nightly: false
+
+```
+
+### Example including [code signing](https://docs.nextcloud.com/server/latest/admin_manual/issues/code_signing.html)
+The following example show how to upload a app which makes use of [code signing](https://docs.nextcloud.com/server/latest/admin_manual/issues/code_signing.html). It assumes the following:
+*  You have a `Makefile` inside of your app's root folder which makes use of the [`occ integrity:sign-app`](https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/occ_command.html#integrity-check)-command to sign the code of your app before creating the app's tarball
+* The `Makefile` will use the app's private key and certificate from the root of the app folder. Please modify the paths if they do not fit your needs.
+
+> :warning: Always make sure you never upload your app's private key inside the app's tarball by accident! Unfortunately the [`occ integrity:sign-app`](https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/occ_command.html#integrity-check)-command currently needs the `.key` and `.crt`-files to be persisted on the local disk for signing.
+
+```yaml
+name: Build and publish app release
+
+on:
+  release:
+      types: [published]
+
+env:
+  APP_NAME: workflow_ocr
+
+jobs:
+  build_and_publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout NC server
+        uses: actions/checkout@v2
+        with:
+          repository: nextcloud/server
+          ref: 'master'
+      - name: Checkout NC submodules
+        shell: bash
+        run: |
+          auth_header="$(git config --local --get http.https://github.com/.extraheader)"
+          git submodule sync --recursive
+          git -c "http.extraheader=$auth_header" -c protocol.version=2 submodule update --init --force --recursive --depth=1
+      - name: Checkout app
+        uses: actions/checkout@v2
+        with:
+          path: ${{ env.APP_NAME }}
+      - name: Run build
+        run: |
+          cd ${{ env.APP_NAME }}
+          echo "${{ secrets.APP_PRIVATE_KEY }}" > ${{ env.APP_NAME }}.key
+          echo "${{ secrets.APP_CERT }}" > ${{ env.APP_NAME }}.crt
+          make appstore
+          rm ${{ env.APP_NAME }}.key
+          rm ${{ env.APP_NAME }}.crt
+      - name: Upload app tarball to release
+        uses: svenstaro/upload-release-action@v2
+        id: attach_to_release
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: ${{ env.APP_NAME }}/build/artifacts/appstore/${{ env.APP_NAME }}.tar.gz
```

Comment:
Not sure all apps follows that path :thinking: 
Krankerl outputs to `build/artifacts/<app_id>.tar.gz`
https://github.com/ChristophWurst/krankerl